pkg/kubelet/container/runtime.go

```diff
@@ -52,6 +52,10 @@ type Runtime interface {
 	// TODO(yifan): Pull/Remove images
 }
 
+type ContainerRunner interface {
```

Comment:
@pmorie It's planned to be implemented by docker and rocket. I did a rocket implementation using these interfaces. Now I am trying to refactoring the docker code to make it close to these interfaces. The interface is not stable yet, it's meant to be modified.